modules/must-gather-flags.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="must-gather-flags_{context}"]
+= Must-gather flags
+
+The flags listed in the following table are available to use with the `oc adm must-gather` command.
+
+.{product-title} flags for `oc adm must-gather`
+[cols="1,1,3",options="header"]
+|====
+|Flag |Example command |Description
+
+|`--all-images`
+|`oc adm must-gather --all-images=false`
+|Collect `must-gather` data using the default image for all Operators on the cluster that are annotated with `operators.openshift.io/must-gather-image`.
+
+|`--dest-dir`
+|`oc adm must-gather --dest-dir='<directory_name>'`
+|Set a specific directory on the local machine to where the gathered data will be written.
```

Comment:
```suggestion
|Set a specific directory on the local machine where the gathered data is written.
```
I feel like you are trying to avoid ending with a preposition in this case. However, the [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=grammar-prepositions) allows for prepositions at the end of sentences in order to avoid awkward and stilted language.